It is assumed that the mean systolic blood pressure is **μ = 120 mm Hg**. In the Honolulu Heart Study, a sample of **n = 100 people** had an **average systolic blood pressure of 130.1 mm Hg** with a **standard deviation of 21.21 mm Hg**. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

#### Set up the hypothesis test.
#### Write down all the steps followed for setting up the test.

**H0:** mean systolic blood pressure=> μ = 120 mm Hg
**H1:** mean systolic blood pressure > 120 mm Hg


**Sample= 100**
Sample's mean systolic blood pressure:130.1
Sample's standard deviation: 21.21 mm Hg


#### Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

In [5]:
import math

sample_mean = 130.1 
pop_mean = 120
sample_std = 21.21
n = 100

statistic = (sample_mean - pop_mean)/(sample_std/math.sqrt(n))
statistic

4.761904761904759

In [6]:
from scipy.stats import t

p_value = t.sf(abs(statistic), n-1) 
p_value

3.2813509086043083e-06

In [9]:
statistic = (sample_mean - pop_mean)/(sample_std/math.sqrt(n))
statistic

4.761904761904759

P_values is lesss than 0,05, so the probabaility of H0 is less than 0,05 so we reject the Hypothesis that the mean systolic blood pressure is equal to 120 mm Hg.

In [8]:
#Critical Value

import scipy.stats

df=99
q= 1-0.05
scipy.stats.t.ppf(q, df)

1.6603911559963895



Looking at the table, and using the code above we see the critical value for one-tailed alpha 0.05 and degree of freedon (dof) of 99 is 1.660.

We have a right tailes test and statistics is much bigger than the critical value so again the H0 should get rejected.
